input/standardized-verification/bo/actual_sfr_tru

```diff
@@ -0,0 +1,102 @@
+0
```

Comment:
What's the meaning of this file, and why does one need these + signs at the beginning of each line?

Reply:
Those are the results from orion and bo's 'model answers'. I think the plus sign is a way for github to say that those lines were added. I don't see them on the actual file..?